src/seedu/addressbook/data/person/Name.java

```diff
@@ -43,6 +45,10 @@ public static boolean isValidName(String test) {
         return Arrays.asList(fullName.split("\\s+"));
     }
 
+    public List<String> getWordsInLowerCaseName() {
```

Comment:
Function name should be in accordance with the job done that is splitting and converting to lowercase